platform/ATCmdParser.cpp

```diff
@@ -102,7 +107,7 @@ int ATCmdParser::read(char *data, int size)
 
 
 // printf/scanf handling
-int ATCmdParser::vprintf(const char *format, va_list args)
+int ATCmdParser::vprintf(const char *format, std::va_list args)
```

Comment:
std::vaargs types are in `#include <cstdarg>` How is it included ?

Reply:
https://github.com/ARMmbed/mbed-os/pull/7864/files/1d23843833c19d913601f40e34ef3880f0a3a77c#diff-61ea3747ffa004896edb10ecbdcdc1a0R23